desc/io/optimizable_io.py

```diff
@@ -86,7 +86,10 @@ def _unjittable(x):
     if isinstance(x, dict):
         return all([_unjittable(y) or y is None for y in x.values()])
     if hasattr(x, "dtype") and np.ndim(x) == 0:
-        return np.issubdtype(x.dtype, np.bool_) or np.issubdtype(x.dtype, np.int_)
+        try:
+            return np.issubdtype(x.dtype, np.bool_) or np.issubdtype(x.dtype, np.int_)
```

Comment:
this change was, if I recall correctly, to handle setting the `key` in `stochastic` dict to be static, I dont remember exactly the issue but I think that whatever type of object the JAX key is does not have a `.dtype` attribute, maybe